$OpenBSD: patch-src_extensions-tool_meson_build,v 1.1 2019/11/02 15:04:09 jasper Exp $

XXX: disable manpages for now due to:
/usr/local/bin/a2x -D src/extensions-tool/man [...]
a2x: WARNING: --destination-dir option is only applicable to HTML based outputs

Index: src/extensions-tool/meson.build
--- src/extensions-tool/meson.build.orig
+++ src/extensions-tool/meson.build
@@ -39,6 +39,6 @@ if bash_completion.found()
   )
 endif
 
-if get_option('man')
-  subdir('man')
-endif
+#if get_option('man')
+#  subdir('man')
+#endif
